src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor

```diff
@@ -100,7 +101,7 @@
                 {
                     <button class="@GetLinkClass(navLinkItem)"
                             title="@(navLinkItem.Title.HasValue() ? navLinkItem.Title : navLinkItem.Name)"
-                            style="padding: 0px 20px 0px @(((navLinkItem.Depth * 14)+27).ToString())px;"
+                            style="padding: 0px 20px 0px @(((navLinkItem.Depth * 14) + 27).ToString())px;"
```

Comment:
please remove the px unit where the value of the CSS property is equal to 0.

Reply:
done